src/assets/stylesheets/avatar.scss

```diff
@@ -0,0 +1,96 @@
+@import 'shared';
+
+:local(.avatar-landing) {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  pointer-events: auto;
+
+  @extend %default-font;
+  @extend %unselectable;
+
+  :local(.logo) {
+    width: 150px;
+    position: absolute;
+    right: 32px;
+    bottom: 32px;
+    display: none;
+
+    @media (min-width: 500px) {
+      display: block;
```

Comment:
I think it's important that we fit the logo on the page even on mobile. Unlike the Scene page, this page makes very little sense without context, if you've never heard of Hubs.